### ISSUE TITLE
Segfault/Illegal instruction on Intel iGPUs on macOS

### ISSUE THREAD

**Issue by YuriNator557** [RPCS3/rpcs3#12404]
### Quick summary

Ever since macOS support was added, we have had several users report games not starting on their Intel Macs due to this problem. It seems to affect all games, there is no commonality with the games that people report aren't working.

### Attach a log file

[error-Banana737.zip](https://github.com/RPCS3/rpcs3/files/9201207/error-Banana737.zip)
[RPCS3(1).log](https://github.com/RPCS3/rpcs3/files/9201215/RPCS3.1.log)
[RPCS3(2).log](https://github.com/RPCS3/rpcs3/files/9201221/RPCS3.2.log)

Assorted logs from some previous reports

### System configuration

Seems to affect any Intel iGPU whether it be the newer Iris Xe graphics or older Iris Pro/Plus graphics. CPU doesn't seem to make a difference.

**Comment by digant73** [comment]
@YuriNator557 is that still an issue?

**Comment by corruptedcervid** [comment]
No, the issue still occurs.

**Comment by pierreluc16** [comment]
> No, the issue still occurs.

The issue still occurs on 30 May 2026 ?

**Comment by kd-11** [comment]
Definitely not our bug on this one. Also didn't apple discontinue intel CPUs in 2020 before Xe became a thing?

**Comment by kd-11** [comment]
This issue just needs a lldb session to confirm which instruction is triggering the crash, then we report the bug to whoever owns said module and close it on our end. I don't have an intel-based mac anymore though.
@schm1dtmac Do you have access to this platform?

**Comment by schm1dtmac** [comment]
> This issue just needs a lldb session to confirm which instruction is triggering the crash, then we report the bug to whoever owns said module and close it on our end. I don't have an intel-based mac anymore though. [@schm1dtmac](https://github.com/schm1dtmac) Do you have access to this platform?

Nope, I doubt most people do either given that Apple mostly ditched Intel iGPU based Macs around 2020 with the M1 MacBook Air/MacBook Pro (pierreluc’s comment isn’t relevant to the issue as that only pertained to the Mac Pro, which never used Intel GPUs).

**Comment by Valtekken** [comment]
> > This issue just needs a lldb session to confirm which instruction is triggering the crash, then we report the bug to whoever owns said module and close it on our end. I don't have an intel-based mac anymore though. [@schm1dtmac](https://github.com/schm1dtmac) Do you have access to this platform?
> 
> Nope, I doubt most people do either given that Apple mostly ditched Intel iGPU based Macs around 2020 with the M1 MacBook Air/MacBook Pro (pierreluc’s comment isn’t relevant to the issue as that only pertained to the Mac Pro, which never used Intel GPUs).

I have a 2012 Mac Mini, maybe I can help with this?

**Comment by corruptedcervid** [comment]
> > > This issue just needs a lldb session to confirm which instruction is triggering the crash, then we report the bug to whoever owns said module and close it on our end. I don't have an intel-based mac anymore though. [@schm1dtmac](https://github.com/schm1dtmac) Do you have access to this platform?
> > 
> > 
> > Nope, I doubt most people do either given that Apple mostly ditched Intel iGPU based Macs around 2020 with the M1 MacBook Air/MacBook Pro (pierreluc’s comment isn’t relevant to the issue as that only pertained to the Mac Pro, which never used Intel GPUs).
> 
> I have a 2012 Mac Mini, maybe I can help with this?

The 2012 one is almost certain to be below minimum specs.

**Comment by Valtekken** [comment]
> > > > This issue just needs a lldb session to confirm which instruction is triggering the crash, then we report the bug to whoever owns said module and close it on our end. I don't have an intel-based mac anymore though. [@schm1dtmac](https://github.com/schm1dtmac) Do you have access to this platform?
> > > 
> > > 
> > > Nope, I doubt most people do either given that Apple mostly ditched Intel iGPU based Macs around 2020 with the M1 MacBook Air/MacBook Pro (pierreluc’s comment isn’t relevant to the issue as that only pertained to the Mac Pro, which never used Intel GPUs).
> > 
> > 
> > I have a 2012 Mac Mini, maybe I can help with this?
> 
> The 2012 one is almost certain to be below minimum specs.

Welp, I tried

**Comment by kd-11** [comment]
Intel GPUs as a whole are not fully supported by the emulator due to the myriad of hardware and software limitations on that platform. I don't see any reason to keep this issue open while taking that into consideration.
